### ISSUE TITLE
ImportError: No module named 'bulk_ad_group_negative_audience_association'

### ISSUE THREAD

**Issue by jfscyberpub** [BingAds/BingAds-Python-SDK#110]
For no reason, I got this error:

```
  File "/home/jonathan/projects/sites/tool/project/bing_ads/core.py", line 7, in <module>
    from bingads.v12.bulk import BulkServiceManager
  File "/home/jonathan/projects/venvs/tool/lib/python3.5/site-packages/bingads/v12/bulk/__init__.py", line 11, in <module>
    from .file_reader import *
  File "/home/jonathan/projects/venvs/tool/lib/python3.5/site-packages/bingads/v12/bulk/file_reader.py", line 2, in <module>
    from .entities.bulk_entity import BulkEntity
  File "/home/jonathan/projects/venvs/tool/lib/python3.5/site-packages/bingads/v12/bulk/entities/__init__.py", line 25, in <module>
    from .audiences import *
  File "/home/jonathan/projects/venvs/tool/lib/python3.5/site-packages/bingads/v12/bulk/entities/audiences/__init__.py", line 10, in <module>
    from .bulk_ad_group_negative_product_audience_association import *
  File "/home/jonathan/projects/venvs/tool/lib/python3.5/site-packages/bingads/v12/bulk/entities/audiences/bulk_ad_group_negative_product_audience_association.py", line 1, in <module>
    from bulk_ad_group_negative_audience_association import *
ImportError: No module named 'bulk_ad_group_negative_audience_association'
```
I have `bingads==12.0.3`

It works fine with `bingads==12.0.2`

Please advise

**Comment by c0bra** [comment]
Pretty shocking to have our production code suddenly break due to a patch-level update.

**Comment by jfscyberpub** [comment]
Thanks for your answer. Do you have a release date for this issue ?

**Comment by c0bra** [comment]
@jfscyberpub sorry to be confusing. I'm using this library like you; I'm not one of the developers.

**Comment by jfscyberpub** [comment]
@c0bra my bad. According to their past releases, the next one should be scheduled for February 10th but it seems there is no guarantee that they'll fix this one.

**Comment by qitia** [comment]
Thanks for reporting this. I will check and update.

**Comment by jfscyberpub** [comment]
@qitia Thank you for the feedback.

**Comment by qitia** [comment]
Hot fix version [12.0.3.1](https://github.com/BingAds/BingAds-Python-SDK/releases/tag/v12.0.3.1) published for this issue. Please try it out. @jfscyberpub

**Comment by jfscyberpub** [comment]
@qitia It works. Thanks a lot.
